concepts/05 UI Components/SelectBox/How to Choose a Drop-Down Editor.md

```diff
@@ -0,0 +1,89 @@
+        <td><a href="/Documentation/Guide/UI_Components/TagBox/Keyboard_Support/">Yes</a></td>
+        <td><a href="/Documentation/Guide/UI_Components/Lookup/Keyboard_Support/">Yes</a></td>
+        <td>Yes</td>
+        <td>Requires custom configuration</td>
+    </tr>
+    <tr>
+        <td>Search box in a drop-down field</td>
+        <td>-</td>
+        <td>-</td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#searchEnabled">Yes</a></td>
+        <td>-</td>
+        <td>Requires custom configuration</td>
+    </tr>
+    <tr>
+        <td>Display groups</td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#grouped">Yes</a></td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#grouped">Yes</a></td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#grouped">Yes</a></td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#grouped">Yes</a></td>
+        <td>Requires custom configuration</td>
+    </tr>
+    <tr>
+        <td>Search based on user input</td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#minSearchLength">Yes</a></td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#minSearchLength">Yes</a></td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#minSearchLength">Yes</a></td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#minSearchLength">Yes</a></td>
+        <td>Requires custom configuration</td>
+    </tr>
+    <tr>
+        <td>Multiple selection</td>
+        <td>-</td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#showSelectionControls">Yes</a></td>
+        <td>-</td>
+        <td>-</td>
+        <td>Requires custom configuration</td>
+    </tr>
+    <tr>
+        <td>Paging</td>
+        <td><a href="/Documentation/Guide/UI_Components/SelectBox/Enable_Paging/">Yes</a></td>
+        <td><a href="/Documentation/Guide/UI_Components/TagBox/Enable_Paging/">Yes</a></td>
+        <td><a href="/Documentation/Guide/UI_Components/Lookup/Enable_Paging/">Yes</a></td>
+        <td>-</td>
+        <td>Requires custom configuration</a></td>
+    </tr>
+</table>
+
+Unlike the rest of the editors, DropDownBox allows you to put any content in its drop-down field. This feature makes it more versatile but at the same time requires more effort from the developer. We recommend using DropDownBox in those advanced use cases that cannot be satisfied by other editors. Refer to the DropDownBox [help topics](/Documentation/Guide/UI_Components/DropDownBox/Overview/) and [demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/) for further information.
```

Comment:
DropDownBox allows you to put any content in its drop-down field. This feature makes it more versatile but it is also more complex to implement. 

We recommend that you use DropDownBox in cases **that cannot be satisfied by other editors** - Why do you use this phrase? How are they not **satisfied** by other editors? Which cases do you refer to?

Reply:
Hello. The DDBox is basically a container for other components, and this allows users implement almost any scenario they can imagine. For instance, they want display two columns with text and/or images in the drop-down field. They can nest our dxDataGrid component into the drop-down field, and our DDBox is the only component among other drop-down editors that has such capabilities.

Reply:
I think a detailed explanation, that is, something like 'DropDownBox allows you to put any content in its drop-down field. This feature makes it more versatile but it is also more complex to implement' (+ perhaps additional information) already explains why you would use this instead of other editors. You can even incorporate an example of such scenario in the description to help illustrate what makes it more versatile.